examples/react-app/backend/src/index.js

```diff
@@ -26,7 +26,14 @@ const sequelize = new Sequelize(process.env.DB_ADDR, {
       Sequelize.ConnectionError,
       Sequelize.ConnectionRefusedError
     ],
-  }
+  },
+  ssl: process.env.DB_USE_SSL === 'true',
```

Comment:
I was wrong on this. We should pass in a ca like you were before:
https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem

```ts
ssl: process.env.DB_CA ? { ca: process.env.DB_CA } : undefined
```